resin/models/device.py

```diff
@@ -531,12 +532,12 @@ def register(self, app_id, uuid):
             uuid (str): device uuid.
 
         Returns:
-            str: dictionary contains device info (can be parsed to dict).
+            dict: dictionary contains device info (can be parsed to dict).
```

Comment:
You can get rid of the " (can be parsed to dict)" here now :)

Reply:
Updated!

I fixed the single-quotes comment as well.